sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -85,102 +85,98 @@ client = FormRecognizerClient(endpoint, credential)
 ## Key concepts
 
 ### FormRecognizerClient
-A `FormRecognizerClient` is the Form Recognizer interface to use for analyzing receipts and extracting layout items from 
-forms. It provides a different method based on input from a URL or input from a stream. 
-Use the receipt methods to extract receipt field values and locations from receipts from the United States from a URL
-or stream. Use the layout methods to extract table data and geometry from a URL or stream.
-
-### CustomFormClient
-A `CustomFormClient` is the Form Recognizer interface to use for creating, using, and managing custom machine-learned models. 
-It provides operations for training models on forms you provide, and extracting field values and locations from your 
-custom forms.  It also provides operations for viewing and deleting models, as well as understanding how close you 
+A `FormRecognizerClient` is the Form Recognizer interface to use for analyzing receipts and extracting layout items from
+forms. It provides different methods based on inputs from a URL and inputs from a stream. You can extract receipt field
+values from receipts from the US, table data and geometry from a URl or stream, and use a custom trained model to extract
+form data.
+
+### FormTrainingClient
+A `FormTrainingClient` is the Form Recognizer interface to use for creating, using, and managing custom machine-learned models.
+It provides operations for training models on forms you provide, and extracting field values and locations from your
```

Comment:
locations -> text placement? bounding boxes?

Reply:
I like text placement, it's less specific than bounding boxes